backend/JDISCTF/api/admin/events.py

```diff
@@ -0,0 +1,136 @@
+"""Categories routes"""
+import flask_rebar
+from flask_rebar import errors
+
+from JDISCTF.app import DB, REGISTRY
+from JDISCTF.models import Administrator, Event
+from JDISCTF.permission_wrappers import require_admin, require_admin_for_event
+from JDISCTF.schemas import GenericMessageSchema
+from JDISCTF.schemas.admin import AdminEventListSchema, AdminEventRequestSchema, AdminEventSchema
+
+
+@REGISTRY.handles(
+    rule="/admin/event/all",
+    method="GET",
+    response_body_schema=AdminEventListSchema(many=True)
+)
+@require_admin
+def get_admin_events(current_admin: Administrator):
```

Comment:
Pourquoi ne pas mettre _ à la place de `current_admin`?

Reply:
Ça éviterait le comment de pylint en plus

Reply:
parce que current_admin est injecté par @require_admin. Ça plante si c'est pas appelé current_admin.

Reply:
Ah right. Well that kinda sucks I guess :thinking: